fmlonly/src/main/java/net/minecraftforge/fmlonlyclient/ClientModLoader.java

```diff
@@ -111,78 +99,13 @@ private static void finishModLoading(ModWorkManager.DrivenExecutor syncExecutor,
 
     public static boolean completeModLoading()
     {
-//        RenderSystem.disableTexture();
-//        RenderSystem.enableTexture();
-//        List<ModLoadingWarning> warnings = ModLoader.get().getWarnings();
-//        boolean showWarnings = true;
-//        try {
-//            showWarnings = ForgeConfig.CLIENT.showLoadWarnings.get();
-//        } catch (NullPointerException e) {
-//            // We're in an early error state, config is not available. Assume true.
-//        }
-//        if (!showWarnings) {
-//            //User disabled warning screen, as least log them
-//            if (!warnings.isEmpty()) {
-//                LOGGER.warn(LOADING, "Mods loaded with {} warning(s)", warnings.size());
-//                warnings.forEach(warning -> LOGGER.warn(LOADING, warning.formatToString()));
-//            }
-//            warnings = Collections.emptyList(); //Clear warnings, as the user does not want to see them
-//        }
-//        File dumpedLocation = null;
-//        if (error == null) {
-//            // We can finally start the forge eventbus up
-//            MinecraftForge.EVENT_BUS.start();
-//        } else {
-//            // Double check we have the langs loaded for forge
-//            LanguageHook.loadForgeAndMCLangs();
-//            dumpedLocation = CrashReportExtender.dumpModLoadingCrashReport(LOGGER, error, mc.gameDirectory);
-//        }
-//        if (error != null || !warnings.isEmpty()) {
-//            mc.setScreen(new LoadingErrorScreen(error, warnings, dumpedLocation));
-//            return true;
-//        } else {
-//            ClientHooks.logMissingTextureErrors();
-//            return false;
-//        }
         return true;
```

Comment:
Is the static method just returning true?
Either remove the method or document why it always returns the same value.